lib/assignDisabledRanges.js

```diff
@@ -16,11 +16,20 @@ const ALL_RULES = "all"
   }>
 }*/
 
+/*:: type returningResult = {
+  stylelint: {
+    disabledRanges: disabledRangeObject,
+    ruleSeverities?: Object,
+    customMessages?: Object,
+    quiet?: boolean,
+  }
+}*/
+
 // Run it like a plugin ...
 module.exports = function (
   root/*: Object*/,
   result/*: Object*/
-) {
+)/*: returningResult*/ {
```

Comment:
@davidtheclark do you mean I have to rename this type?
There are just several points:

- I did not find `postcss$result` in declaration files, that's why I assumed that you've meant renaming
- I've run tests and found out that returning object is much bigger than I described. Should I expand the type annotation of the returning object?